app/web/src/features/dashboard/DashboardBanners.tsx

```diff
@@ -1,12 +1,12 @@
-import { Typography } from "@material-ui/core";
+import { Button as MuiButton, Typography } from "@material-ui/core";
 import { Alert as MuiAlert } from "@material-ui/lab/";
 import Alert from "components/Alert";
-import StyledLink from "components/StyledLink";
 import { Error as GrpcError } from "grpc-web";
 import { GetAccountInfoRes } from "proto/account_pb";
 import { accountInfoQueryKey } from "queryKeys";
 import React from "react";
 import { useQuery } from "react-query";
+import { Link as RouterLink } from "react-router-dom";
```

Comment:
`Link` is fine - usually we import the MUI link as `MuiLink` but since we added `StyledLink` recently we shouldn't need to do that either